app crashes when voting on proposal

Clicking the "Vote" button crashes the app

<img width="360" alt="Screenshot 2023-08-28 at 3 38 02 PM" src="https://github.com/crypto-power/cryptopower/assets/25265396/49125c4f-935e-4845-b83f-55adc6ca23f5">


```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x8 pc=0x102cc8360]

goroutine 30 [running]:
github.com/crypto-power/cryptopower/ui/page/governance.(*voteModal).sendVotes.func1(...)
        /Users/kennedy/go/src/github.com/crypto-power/cryptopower/ui/page/governance/proposal_vote_modal.go:137
github.com/crypto-power/cryptopower/ui/page/governance.(*voteModal).sendVotes(0x14002590000)
        /Users/kennedy/go/src/github.com/crypto-power/cryptopower/ui/page/governance/proposal_vote_modal.go:145 +0x1b0
github.com/crypto-power/cryptopower/ui/page/governance.(*voteModal).Handle(0x14002590000)
        /Users/kennedy/go/src/github.com/crypto-power/cryptopower/ui/page/governance/proposal_vote_modal.go:198 +0x174
github.com/crypto-power/cryptopower/ui.(*Window).handleFrameEvent(0x14002622140, {{0xc1333d54b55a9d78, 0x11105d4c2a, 0x104444820}, {0x40000000, 0x40000000}, {0x640, 0x4b0}, {0x0, 0x0, ...}, ...})
        /Users/kennedy/go/src/github.com/crypto-power/cryptopower/ui/window.go:231 +0xf0
github.com/crypto-power/cryptopower/ui.(*Window).HandleEvents(0x14002622140)
        /Users/kennedy/go/src/github.com/crypto-power/cryptopower/ui/window.go:202 +0x1c0
main.main.func3()
        /Users/kennedy/go/src/github.com/crypto-power/cryptopower/main.go:110 +0x20
created by main.main in goroutine 1
        /Users/kennedy/go/src/github.com/crypto-power/cryptopower/main.go:108 +0x4a4
exit status 2
```

--- a/libwallet/politeia.go
+++ b/libwallet/politeia.go
@@ -65,16 +65,24 @@ type ProposalVote struct {
 	politeia.ProposalVote
 }
 
-func ConvertVotes(votes []*ProposalVote) []*politeia.ProposalVote {
-	var eligibleTickets = make([]*politeia.ProposalVote, len(votes))
-	for i, value := range votes {
-		eligibleTickets[i] = &politeia.ProposalVote{
-			Bit: value.Bit,
+// WrapVote, wraps vote type of politeia.ProposalVote into libwallet.ProposalVote
+func WrapVote(hash, address, bit string) *ProposalVote {
+	return &ProposalVote{
+		ProposalVote: politeia.ProposalVote{
+			Bit: bit,
 			Ticket: &politeia.EligibleTicket{
-				Hash:    value.Ticket.Hash,
-				Address: value.Ticket.Address,
+				Hash:    hash,
+				Address: address,
 			},
-		}
+		},
+	}
+}
+
+func ConvertVotes(votes []*ProposalVote) []*politeia.ProposalVote {
+	eligibleTickets := make([]*politeia.ProposalVote, len(votes))
+	for i, value := range votes {
+		eligibleTickets[i] = &WrapVote(value.Ticket.Hash,
+			value.Ticket.Address, value.Bit).ProposalVote
 	}
 	return eligibleTickets
 }

--- a/ui/page/governance/proposal_vote_modal.go
+++ b/ui/page/governance/proposal_vote_modal.go
@@ -129,16 +129,7 @@ func (vm *voteModal) sendVotes() {
 	votes := make([]*libwallet.ProposalVote, 0)
 	addVotes := func(bit string, count int) {
 		for i := 0; i < count; i++ {
-
-			// get and pop
-			tickets = tickets[1:]
-
-			vote := &libwallet.ProposalVote{}
-			vote.Ticket.Hash = tickets[0].Hash
-			vote.Ticket.Address = tickets[0].Address
-			vote.Bit = bit
-
-			votes = append(votes, vote)
+			votes = append(votes, libwallet.WrapVote(tickets[i].Hash, tickets[i].Address, bit))
 		}
 	}
 
@@ -159,7 +150,7 @@ func (vm *voteModal) sendVotes() {
 				pm.SetLoading(false)
 				return false
 			}
-			pm.Dismiss()
+			vm.Dismiss()
 			infoModal := modal.NewSuccessModal(vm.Load, values.String(values.StrVoteSent), modal.DefaultClickFunc())
 			vm.ParentWindow().ShowModal(infoModal)
 			go vm.WL.AssetsManager.Politeia.Sync(ctx)

